app/src/main/res/values/strings.xml

```diff
@@ -868,9 +868,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="team_mode">Team Mode</string>
     <string name="team_mode_exit">Exit Team Mode</string>
     <string name="team_mode_description">When mapping the same area together in a group, you can enter team mode to split up quests between all team members.</string>
-    <string name="team_mode_team_size_label">How many people are there?</string>
+    <string name="team_mode_team_size_label2">How many people are mapping? Enter the same number on everyone's phone.</string>
     <string name="team_mode_team_size_hint">Team mode works for groups of 2 to 12 people.</string>
-    <string name="team_mode_choose_color2">Now everybody in your group chooses a unique color on their phone:</string>
+    <string name="team_mode_choose_color2">Next, each person must pick a different color. Choose yours:</string>
```

Comment:
I went back and forth a lot, about which of the two strings should be "everyone" vs "each person". I think maybe they should be swapped. @peternewman maybe you could give a second opinion?

Reply:
Nothing strong here:
https://www.grammarly.com/blog/everyone-vs-every-one/

There are more hits for "everyone must pick" rather than "each person must pick" (~1.5x), also I'd broadly rate the first page of hits for the former better than the latter.

Although a lot more "on everyone's phone" than "on each person's phone" (> 10x), although I'd broadly rate the latter higher.

I'm guessing we don't want everyone/everybody for both? :laughing: 

In which case I'd probably just favour how it currently is, as the colour is the thing we want to be distinct, and each person feels more directed to the individual.

So now we've got two opinions, we need a third to decide! :rofl:

Reply:
> I'm guessing we don't want everyone/everybody for both?

Yeah, I was trying not to use the same word twice in a row. I'd be fine with everyone/everybody.

> the colour is the thing we want to be distinct, and each person feels more directed to the individual

That's what I was going for.

> So now we've got two opinions, we need a third to decide! :rofl:

Ok— If we don't care about including "phone", I'd be happy changing the first to, "How many people are mapping? Everyone should enter the same number." Or "They must enter the same number", if we're going for brevity.

Then I'd be happy to switch to "Next, everyone must pick a different color". Otherwise, I want to stick with "everyone's phone" for brevity.

Reply:
But, at this point I'm happy *enough* with how it is. If we all are, I'm happy not to bother perfecting the phrasing.